account_analytic_default_account/models/account_analytic_default_account.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticDefaultAccount(models.Model):
+    _inherit = "account.analytic.default"
+
+    account_id = fields.Many2one(
+        'account.account', string='Account',
+        ondelete='cascade',
+        help="Select an account which will use analytic account specified in "
```

Comment:
can you rephrase this? The way I get is: "This account will be used by default for sale orders or invoices" or something similar :)